### PR TITLE
Isochrone : add fields in the response

### PR DESCRIPTION
**Pull request by ZoeBrunet**
Needs PR  Isochrone : change response in navitia-proto #61

### CODE REVIEW

File: source/navitia-proto [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit fe918b031d8ec7f74065c3fcf82921b2f1378cda
+Subproject commit 89cb129d4bbcd2ade44d580c1472fc73e9635dcd
```
Comment (pbougue) [inline source/navitia-proto]: to be updated when merged

File: source/routing/raptor_api.cpp [hunk 2]
```diff
@@ -1174,14 +1174,26 @@ static void print_polygon(std::stringstream& os, const type::Polygon& polygon) {
     os << "]";
 }
 
-static void add_graphical_isochrone(const type::MultiPolygon& shape, PbCreator& pb_creator) {
+static void add_graphical_isochrone(const type::MultiPolygon& shape,
+                                    const int& min_duration,
+                                    const int& max_duration,
+                                    PbCreator& pb_creator,
+                                    type::EntryPoint origin,
```
Comment (pbougue) [inline source/routing/raptor_api.cpp]: in this context, it may be clearer to rename it "center"

File: source/routing/raptor_api.cpp [hunk 3]
```diff
@@ -1223,7 +1235,7 @@ pbnavitia::Response make_graphical_isochrone(RAPTOR &raptor,
     type::GeographicalCoord coord_origin = origin.coordinates;
     type::MultiPolygon isochrone = build_isochrones(raptor, clockwise, coord_origin, bound_max, bound_min, departures,
                                                     speed, max_duration, min_duration);
-    add_graphical_isochrone(isochrone, pb_creator);
+    add_graphical_isochrone(isochrone, min_duration, max_duration, pb_creator, origin, clockwise);
```
Comment (pbougue) [inline source/routing/raptor_api.cpp]: "center" here also if agreed above

**Comment by pbougue** [comment]
ok for me

File: source/jormungandr/jormungandr/interfaces/v1/GraphicalIsochrone.py [hunk 4]
```diff
@@ -49,9 +49,18 @@
 from jormungandr.interfaces.parsers import unsigned_integer
 from jormungandr.interfaces.v1.journey_common import JourneyCommon, dt_represents, compute_possible_region
 
+class section_place(PbField):
+
+    def output(self, key, obj):
+
+        return super(PbField, self).output(key, obj)
```
Comment (TeXitoi) [inline source/jormungandr/jormungandr/interfaces/v1/GraphicalIsochrone.py]: useless?

File: source/jormungandr/jormungandr/interfaces/v1/Journeys.py [hunk 5]
```diff
@@ -29,11 +29,14 @@
...
 # www.navitia.io
 from __future__ import absolute_import, print_function, unicode_literals, division
+from functools import cmp_to_key
 import logging
 from flask import request, g
 from flask.ext.restful import fields, reqparse, marshal_with, abort
 from flask.ext.restful.inputs import boolean
 from jormungandr import i_manager, app
+from jormungandr.exceptions import RegionNotFound
+from jormungandr.instance_manager import instances_comparator
```
Comment (TeXitoi) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py]: you really need these imports?

File: source/jormungandr/tests/check_utils.py [hunk 6]
```diff
@@ -517,6 +528,8 @@ def is_valid_graphical_isochrone(isochrone, tester, query):
         geojson = g['geojson']
         assert geojson
         is_valid_multipolygon_geojson(geojson)
+        get_valid_unsigned_int(g['max_duration'])
+        get_valid_unsigned_int(g['min_duration'])
```
Comment (TeXitoi) [inline source/jormungandr/tests/check_utils.py:532]: you should also add in `is_valid_graphical_isochrone` that `from` xor `to` is present.

File: source/navitia-proto (hunk 1 above)
Comment (pbougue) [inline source/navitia-proto]: merge done

File: source/jormungandr/tests/check_utils.py (hunk 6 above)
Comment (ZoeBrunet) [inline source/jormungandr/tests/check_utils.py:532]: done

**Comment by TeXitoi** [comment]
OK for me.

**Pull request merged**